dashboard/src/main/webapp/src/router/index.js

```diff
@@ -59,6 +59,12 @@ const routes = [
         name: 'applicationpage',
         component: ApplicationPage,
     },
+    {
+        path: '/nullpage',
```

Comment:
Is this necessary?

Reply:
> Is this necessary?

This is a must to prevent clicking on the drop-down box to jump to the road with links.